hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -246,13 +246,13 @@ private OMConfigKeys() {
   public static final long  OZONE_FS_TRASH_CHECKPOINT_INTERVAL_DEFAULT = 0;
 
   // TODO: Temporary workaround for OM upgrade path and will be replaced once
```

Comment:
Please remove this comment. Probably, you can use the same from my above comment.
```

  // TODO: Temporary workaround for OM upgrade path and will be replaced once
  //  upgrade HDDS-3698 story reaches consensus. Defaulting to 'legacy' so that
  //  existing unit test cases won't be affected. New OM version should be 'prefix'.
```